crates/rolldown/tests/esbuild/loader/loader_file_one_source_two_different_output_paths_css/diff.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  resolver: {
+    "/out/common-LSAMBFUD.png": "assets/common-h8VKzHVD.png",
```

Comment:
I just notice that, if the esbuild's or rolldoen's hash algo is changed, these mappings would be broken and all need to be changed manaully. Should we disable `[hash]` for asset too?

Reply:
The hash algorithm should not be changed frequently, so I think we don't need to do this at this times Or if there are we will generate something like `common$1.png` if there are multiple assets has same basename

Reply:
> The hash algorithm should not be changed frequently

I agree with this. But once it changed, we need to fix all related tests manually. It would be frustrated for people who is touches hash-related code.

Reply:
I will try to support regex match in `esbuild-diff` so hash change, so people should not worry about asset hash algo